pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -288,6 +288,9 @@ func (c *Cluster) UpdateActiveMgrLabel(daemonNameToUpdate string, prevActiveMgr
 			pod.SetLabels(labels)
 			_, err = c.context.Clientset.CoreV1().Pods(c.clusterInfo.Namespace).Update(c.clusterInfo.Context, &pods.Items[i], metav1.UpdateOptions{})
 			if err != nil {
+				// In case of failure we report as 'active manager' the previous value, this way
+				// we force refreshing mgrRoleLabelName label next time this function is called
+				currActiveMgr = prevActiveMgr
 				logger.Infof("cannot update the active mgr pod %q. err=%v", pods.Items[i].Name, err)
```

Comment:
How about changing this to a warning?
```suggestion
				logger.Warningf("cannot update the active mgr pod %q. err=%v", pods.Items[i].Name, err)
```